packages/ddl-shim/src/upgrade.ts

```diff
@@ -10,8 +10,9 @@ interface IDatasourceOutputFilter extends IDatasourceOutput {
     filter: DDL1.IFilter;
 }
 
+const UPGRADE_HEX_CHAR: boolean = false;
```

Comment:
Is the idea here to allow `UPGRADE_HEX_CHAR` to be globally overwritten if/when hex values *should* be upgraded to char?